14_2_elements.js

```diff
@@ -0,0 +1,17 @@
+function byTagName(node, tagName) {
+    const array = [];
+    const elements = node.querySelectorAll(tagName);
+    elements.forEach(element => {
```

Comment:
Instead of creating an array and mutating it, try using `map` - it returns new array :)